CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+v0.8.0
+------
+* Allow code in `drop_features_where` function
+* Add bounding box clipping to exterior boundaries transform.
+* Add normalisation functions for social facilities and medical places.
+* Set default landuse order to 11, and adjust everything else by +1
+* Correct water features such that things above the water should be uniquely above the water
+* Update road sort key values
+* Add csv property matching functions
```

Comment:
* Add csv spreadsheet property matching functions to provide sort_key values on landuse, roads, and other layers.